### ISSUE TITLE
Question About Weight Decay

### ISSUE THREAD

**Issue by HesperX** [zylo117/Yet-Another-EfficientDet-Pytorch#462]
Hi, thank you for this great work.

I find you apply AdamW optimizer with lr=1e-4 and weight_decay=1e-2, which is much lower than the paper.

I know that AdamW needs much lower necessary but I am not sure whether it needs so high weight decay?

**Comment by zylo117** [comment]
Current hps works for most of my custom datasets. For coco, I used another set of hps, which is also written on readme.

**Comment by matzel9** [comment]
Hi ,
first of all, thank you for your great work. I have two questions concerning training coco from scratch. 1. Did you initialize the weights with pretrained image net weights? 2. What do you mean with adapt the hps every few hours. Did you just change the learning rate or also something else?
